src/Squirrel/IUpdateManager.cs

```diff
@@ -172,13 +172,12 @@ public static async Task<ReleaseEntry> UpdateApp(this IUpdateManager This, Actio
                     "Failed to set up uninstaller");
 
                 status("Update was successfully installed. Finishing...");
-            } catch (IOException ex) when ((ex.HResult & 0xFFFF) == 0x27 || (ex.HResult & 0xFFFF) == 0x70) {
+            } catch (Exception ex) when ((ex.HResult & 0xFFFF) == 0x27 || (ex.HResult & 0xFFFF) == 0x70) {
                 // 0x27 (ERROR_HANDLE_DISK_FULL) and 0x70 (ERROR_DISK_FULL) are standard Win32 error codes.
                 // See doc: https://docs.microsoft.com/en-us/openspecs/windows_protocols/ms-erref/18d8fbe8-a967-4f1c-ae50-99ca8e491d2d
-                // Rethrow not enough space exception.
-                throw ex; 
+                throw; 
```

Comment:
Yes, sure. It is a little bit commented in the commit message. It's best practice to use just "throw" without exception because you keep the whole stack trace (which includes original exception stack trace). To dive in deeper please see: https://www.dotnetjalps.com/2013/10/throw-vs-throw-ex-csharp.html.